Pipfile

```diff
@@ -9,38 +9,38 @@ python_version = "3"
 [packages]
 nucypher-core = "==0.13.0"
 # Cryptography
-cryptography = ">=3.2"
+cryptography = "*"
 mnemonic = "*"
 pynacl = ">=1.4.0"
 pyopenssl = "*"
 # Ethereum
-eth-abi = "<5.0.0"  # eth-ape restriction
-eth-tester = "<0.10.0,>0.9.0"  # web3[tester]
-eth-account = "<0.9,>=0.8.0"
+eth-abi = "*"
+eth-tester = "<0.10"
+eth-account = "*"
 eth-utils = "*"
 web3 = ">=6.0.0"
+atxm = ">=0.2.0"
 # Web
 flask = "*"
-hendrix = ">=4.0"
+hendrix = ">=5.0"
 requests = "*"
 mako = "*"
 # CLI
 click = ">=7.0"
 colorama = "*"
 tabulate = "*"
 # Serialization
-bytestring-splitter = ">=2.4.0"
 marshmallow = "*"
-msgpack = "*"
```

Comment:
Note that existing PRE examples use msgpack (although those are broken anyway)

Reply:
`msgpack` is still a project dependency since another dependency depends on it, but it does not need to be pinned.

Reply:
![shipping](https://github.com/nucypher/nucypher/assets/679404/c2a86229-8a4f-48ca-a04d-6d54e3d99048)